Common/TableProducer/PID/pidTPCFullTest.cxx

```diff
@@ -0,0 +1,288 @@
+    for (DeviceSpec device : workflows.devices) {
+      for (auto input : device.inputs) {
+        auto enableFlag = [&input](const std::string particle, Configurable<int>& flag) {
+          const std::string table = "pidTPCFull" + particle;
+          if (input.matcher.binding == table) {
+            if (flag < 0) {
+              flag.value = 1;
+              LOG(info) << "Auto-enabling table: " + table;
+            } else if (flag > 0) {
+              flag.value = 1;
+              LOG(info) << "Table enabled: " + table;
+            } else {
+              LOG(info) << "Table disabled: " + table;
+            }
+          }
+        };
+        enableFlag("El", pidEl);
+        enableFlag("Mu", pidMu);
+        enableFlag("Pi", pidPi);
+        enableFlag("Ka", pidKa);
+        enableFlag("Pr", pidPr);
+        enableFlag("De", pidDe);
+        enableFlag("Tr", pidTr);
+        enableFlag("He", pidHe);
+        enableFlag("Al", pidAl);
+      }
+    }
+
+    //   //    // Getting the parametrization parameters
+    //   //    ccdb->setURL(url.value);
+    //   //    ccdb->setTimestamp(timestamp.value);
+    //   //    ccdb->setCaching(true);
+    //   //    ccdb->setLocalObjectValidityChecking();
+    //   //    // Not later than now objects
+    //   //    ccdb->setCreatedNotAfter(std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count());
+    //   //    //
+    //   //    const std::string fname = paramfile.value;
+    //   //    if (!fname.empty()) { // Loading the parametrization from file
+    //   //      LOG(info) << "Loading exp. TPC response from file" << fname;
+
+    //   //      // LoadResponseFromFile(fname.data());
+
+    //   //      /// This load response is not implemented in the new class. It would be nice if we could load the response object from the file and ccdb. Problem is also the templated class since it would be good to load it once and not for every particle species if possible.
+
+    //   //    } else { // Loading it from CCDB
+    //   //      std::string path = ccdbPath.value;
+    //   //      LOG(info) << "Loading TPC response from CCDB, using path: " << path << " for timestamp " << timestamp.value;
+
+    //   //      // LoadParam(ccdb->getForTimeStamp<Parametrization>(path, timestamp.value));
+    //   //    }
+  }
+
+  void process(Coll const& collisions, Trks const& tracks)
+  {
+
+    o2::pid::tpc::TPCPIDResponse* response;
+
+    const TString fname = paramfile.value;
+
+    if (fname != "") {
+      LOG(info) << "Loading exp. TPC response from file " << fname;
+      try {
+        TFile f(fname, "READ");
+        f.GetObject("TPCPIDResponse", response);
+      } catch (...) {
+        LOG(info) << "Loading the file " << fname << " failed!";
+      };
+    } else {
+      LOG(info) << "No external file specified! Load failed!";
+    };
+
+    // LOG(info) << "Custom TPCPID: MIP value from object is " << response->GetMIP();
```

Comment:
This should go to the init function. See suggestion above. Like this it will load the parametrization for each event I think.
```suggestion
```